docs/API.md

```diff
@@ -588,4 +594,37 @@ Parameters:
   * bounceType: The bounce type ([`'Permanent'`, `'Transient'`, `'Complaint'`])
   * bounceSubType: The bounce sub type string
 
-(Ends)
+## .createSigninCode(code, uid, createdAt)
+
+Create a user-specific, time-limited, single-use code
+that can be used for expedited sign-in.
+
+Parameters:
+
+* `code` (Buffer6):
+  The value of the code
+* `uid` (Buffer16):
+  The uid for the relevant user
+* `createdAt` (number):
+  Creation timestamp for the code, milliseconds since the epoch
+
+## .useSigninCode(code)
+
+Use (and delete) a sign-in code.
```

Comment:
FWIW I was coming here to leave the exact same suggestion, "consume" seems like a helpful verb to make it obvious what happens here.